src/modules/classAwesome.js

```diff
@@ -0,0 +1,63 @@
+export default class Awesome {
+  constructor() {
+    this.data = {};
+    this.record = [];
+  }
+
+  addRecord(chore) {
+    this.data = { chore };
+    this.record.push(this.data);
+    return this.data;
+  }
+
+  // removeRecord(rec) {
+  //   const result = this.record.filter(el => el === rec);
+  // }
+
+  local() {
+    localStorage.setItem('tasks', JSON.stringify(this.record));
+    this.returnInfo();
+  }
+
+  updateInput(value, element) {
+    this.record = JSON.parse(localStorage.getItem('tasks'));
+    this.record.forEach((elementForChore) => {
+      if (elementForChore.chore === element) {
+        elementForChore.chore = value;
+      }
+      localStorage.setItem('tasks', JSON.stringify(this.record));
+    });
+  }
+
+  returnInfo() {
+    const tasks = document.querySelector('#tasks');
+    tasks.innerHTML = '';
+    this.record = JSON.parse(localStorage.getItem('tasks'));
+    if (this.record === null) {
+      this.record = [];
+    } else {
+      let index = 0;
+      this.record.forEach((element) => {
+        index += 1;
+        tasks.innerHTML += `
+        <div id = "organize"${index}>
```

Comment:
[OPTIONAL] the `forEach` function already provides you with an index that increments on every iteration so you don't have to create new variable;
```suggestion
      this.record.forEach((element, index) => {
        tasks.innerHTML += `
        <div id = "organize"${index+1}>
```